articles/active-directory/develop/quickstart-v2-javascript.md

```diff
@@ -117,6 +114,32 @@ Select the option that's suitable to your development environment:
 > > To find the values of **Application (client) ID**, **Directory (tenant) ID**, and **Supported account types**, go to the app's **Overview** page in the Azure portal.
 >
 
+> [!div renderon="docs"]
+> Then, still in the same folder, edit *graphConfig.js* file to set the `graphMeEndpoint` and `graphMeEndpoint` for the `msalConfig` object.
+
+> [!div class="sxs-lookup" renderon="portal"]
```

Comment:
Ln 120-121 Should be taken care with apps.json in the code sample. No longer needed. @hamiltonha